src/main/java/Duke.java

```diff
@@ -39,7 +40,7 @@ public static void doTask(int index) {
         System.out.println(line);
     }
 
-    public static void level3() {
+    public static void level4() {
```

Comment:
"level4" is not an appropriate name for a method. It should be based on what this function performs.

Reply:
It is better if the method is private because the private attributes are getting updated here which strengthens the encapsulation.